scheduled-execution.mdx

```diff
@@ -1,25 +1,32 @@
 ---
-title: Scheduling test report runs
-description: "Run your test reports in regular intervals"
+title: Scheduling test runs
+description: "Run your tests and get test results in regular intervals"
 icon: "clock"
 ---
 
-If you don't have pipelines but nevertheless want to run your test reports on a regular schedule, you can do that via our scheduling feature in the test report section of the project overview page.
+If you don't have CI pipelines but you would like to run your tests on a regular schedule, you can do that via our scheduling feature in the `test reports` section of the project overview page.
```

Comment:
```suggestion
If you don't have a CI pipeline but you would like to run your tests on a regular schedule, you can do that via our scheduling feature in the `test reports` section of the project overview page.
```

Reply:
I think both is fine, but I have never head of someone (aside from @sinisastanic maybe 😄 ) talk about pipeline(s) plural ;)